packages/registry/vite-plugin.js

```diff
@@ -5,6 +5,48 @@ import { createThemeAddonsLoader } from '@plone/registry/create-theme-loader';
 import { createAddonsStyleLoader } from '@plone/registry/create-addons-styles-loader';
 import { createAddonsLocalesLoader } from '@plone/registry/create-addons-locales-loader';
 
+export default function relativeToAbsoluteImportPlugin(options) {
+  const { packages } = options;
+
+  const entries = Object.entries(packages);
+
+  return {
+    name: 'relative-to-absolute-imports',
+    enforce: 'pre',
+
+    async transform(code, id) {
+      const normalizedId = id.split('?')[0];
+
+      const match = entries.find(([, pkgInfo]) =>
+        normalizedId.startsWith(pkgInfo.basePath),
+      );
+
+      if (!match) return;
+
+      const [, pkgInfo] = match;
+
+      const transformedCode = code.replace(
+        /from\s+['"]((?:\.\.?)\/[^'"]+)['"]/g,
```

Comment:
This catches the following cases:

- `from '../test'`
- `from './test'`

And doesn't catch the following ones which are valid syntax:

- `from '.'`
- `from '..'`

We could change it to

```suggestion
        /from\s+['"]((?:\.\.?)\/?[^'"]*)['"]/g,
```

This would fix the issue but would also catch
- `from './'`
- `from '../'`

I think it could be a good change even if not perfect

Reply:
@pnicolli added suggestion and tests.